llmcode/repomap.py

```
@@ -765,32 +807,1274 @@
         # add a bogus tag at the end so we trip the this_fname != cur_fname...
         dummy_tag = (None,)
         for tag in sorted(tags) + [dummy_tag]:
```

Ruff (0.12.2)

Consider `[*sorted(tags), dummy_tag]` instead of concatenation

Replace with `[*sorted(tags), dummy_tag]`

(RUF005)